django_webapps/scripts/db_population/parsers/common_parsers.py

```diff
@@ -88,11 +106,119 @@ def parse_telegram_name(tg_str: str) -> str | None:
     if match_result is not None and len(match_result.group(0)) == len(tg_str):
         return f"@{match_result.group(0)}"
 
-    raise ValueError(f"tg: {tg_str} is not valid")
+    return None
+
+
+def find_any_phone_number(number_str: str) -> list[str]:
+    number_str = number_str.replace(" ", "")
+    regex = re.compile(r"[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}")
+    return re.findall(regex, number_str)
+
+
+def parse_phone_number(number_str: str) -> str | None:
+    number_str = number_str.replace(" ", "")
+    country_codes_to_test = [("", None)] + [
+        (f"+{code}", region[0])
+        for code, region in phonenumbers.COUNTRY_CODE_TO_REGION_CODE.items()
```

Comment:
Note, even like that it will not parse all and we will have like ~400 invalid numbers, not sure how to parse them manually

Reply:
Can I look at these failing numbers? Maybe I could come up with some idea